src/components/About/About.jsx

```diff
@@ -99,12 +98,12 @@ const About = (props) => {
             <BgImage image={data.modding.childImageSharp.gatsbyImageData}>
               <div className="right-overlay_about_content">
                 <Row>
-                  <Col className="empty-about-content" md="7"></Col>
+                  <Col className="empty-about-content" md="7" />
                   <Col md="5" className="text-white float-right ">
                     <div className="ml-2 mr-4 ">
                       <h3 className="ml-2 mb-3 text-white">Modding API</h3>
                       <p className="ml-2 mr-3 text-justify ">
-                        Terasology's engine uses a whitelisting approach to
+                        Terasology&apos;s engine uses a whitelisting approach to
```

Comment:
I understand the technical reason, but I don't like that it makes the text less readable when looked at in source form...